src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportFactoryTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\TurboSms\Tests;
+
+use Symfony\Component\Notifier\Bridge\TurboSms\TurboSmsTransportFactory;
+use Symfony\Component\Notifier\Test\TransportFactoryTestCase;
+use Symfony\Component\Notifier\Transport\TransportFactoryInterface;
+
+final class TurboSmsTransportFactoryTest extends TransportFactoryTestCase
+{
+    /**
+     * @return TurboSmsTransportFactory
+     */
+    public function createFactory(): TransportFactoryInterface
+    {
+        return new TurboSmsTransportFactory();
+    }
+
+    public function createProvider(): iterable
+    {
+        yield [
+            'turbosms://host.test?from=acme',
+            'turbosms://accountSid:authToken@host.test?from=acme',
+            'turbosms://accountSid:authToken@host.test?from=Symfony Community',
```

Comment:
Please add another yield with an expectation for this test, adding a third one won't work

Reply:
done